src/SourceTerms/Deleptonization/CPU_Src_Deleptonization.cpp

```diff
@@ -40,14 +36,13 @@ void Src_SetGPUFunc_Deleptonization( SrcFunc_t & );
 #endif
 void Src_SetConstMemory_Deleptonization( const double AuxArray_Flt[], const int AuxArray_Int[],
                                          double *&DevPtr_Flt, int *&DevPtr_Int );
-void Src_PassData2GPU_Deleptonization();
 void Src_End_Deleptonization();
 
 #endif
 
 GPU_DEVICE static
-real YeOfRhoFunc( const real DENS_CGS, const real DELEP_RHO1, const real DELEP_RHO2,
-                  const real DELEP_YE1, const real DELEP_YE2, const real DELEP_YEC );
+real yeofrhofunc( const real Dens_CGS, const real Delep_Rho1, const real Delep_Rho2,
```

Comment:
Restore the function name to `YeOfRhoFunc`. Check other places in this file too.

Reply:
I restored the function name to `YeOfRhoFunc`.